index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" property="og:title" content="width=device-width">
+    <title>Website</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  
+  <body>
+  <main id="main-doc">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <nav id="navbar">
+        <ul>
+        <li><header> How to work with HTML , CSS, javascript</header></li>
+        <li><a class="nav-link" href="#introduction">Introduction</a></li>
+        <li><a class="nav-link" href="#prerequisite">prerequisite</a></li>
+        <li><a class="nav-link" href="#html">html</a></li>
+        <li><a class="nav-link" href="#CSS">CSS</a></li>
+        <li><a class="nav-link" href="#javascript">javascript</a></li>
+    </ul>
+    </nav>
```

Comment:
I suggest you use Use `header` tag around the `nav` tag

Reply:
* Kindly explain why you request a change - it might be obvious for you but not for someone else

Reply:
since the website has one navigation bar, it is kindly required to include the main navigation in a `nav` element within the masthead `header` element.